js/groupedBarLine.js

```diff
@@ -0,0 +1,623 @@
+/**
+ * Create barLineChart object
```

Comment:
I dont see the code where you render the grouped bar chart. This file is just a copy paste of the bar line chart